src/EditorFeatures/Test2/GoToDefinition/GoToDefinitionTests.vb

```diff
@@ -150,6 +150,34 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.GoToDefinition
             Test(workspace)
         End Sub
 
+        <WpfFact, Trait(Traits.Feature, Traits.Features.GoToDefinition)>
+        Public Sub TestCSharpLiteralGoToDefinition()
+            Dim workspace =
+<Workspace>
+    <Project Language="C#" CommonReferences="true">
+        <Document>
+            [|int|] x = 1$$23;
```

Comment:
📝 The `[|stuff|]` marks the target of the Go To Definition operation. These tests expect a *declaration* to be marked, but you have a *reference* marked. The hard part here is all literals have types defined in the core library, so you probably can't write the following:

```
namespace System { public struct [|Int32|] { } }
```